vendor/engines/split_accounts/app/services/split_accounts/reports/querier.rb

```diff
@@ -0,0 +1,29 @@
+module SplitAccounts
+
+  module Reports
+
+    class Querier < ::Reports::Querier
+
+      def perform
+        super.map do |order_detail|
+          simulate_order_details(order_detail)
+        end.flatten
+      end
+
+      def simulate_order_details(order_detail)
+        if order_detail.account.is_a?(SplitAccounts::SplitAccount)
```

Comment:
Is it possible to make this check not necessary? Can you call `splits` on a non-split account and have it return `self`? Or a 100% split version of itself? Then the rest of the logic should apply.